README.md

```diff
@@ -125,7 +149,36 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+class Lamp {
+  @wishes_remaining 3
+}
+end
+
+class Genie{
+  @name 'Genie'
+}
+end
+
+class Lamp < ActiveRecord::Base
+end
+
+class Genie < ActiveRecord::Base
+end
```

Comment:
These two ActiveRecord classes are correct class definitions. But the question is asking how you would do everything in that list assuming you already have your tables, models and associations already set up. Really, it's asking you to run a number of ActiveRecord commands.

So the answer to the first question, which asks you to create a lamp and genie, would look something like this...

```
lamp = Lamp.new( wishes_remaining: 3 )
genie = Genie.new( name: "Genie" )
```

Encourage you to tackle the rest of these questions again using ActiveRecord.